client/tests/webdriver/specs/newUserUtils.js

```diff
@@ -0,0 +1,34 @@
+import moment from "moment"
+import CreatePerson from "../pages/createNewPerson.page"
+import Home from "../pages/home.page"
+import OnboardPage from "../pages/onboard.page"
+
+// Only required fields in onboarding/Edit page
+// No lastName, status, role
+export const examplePersonDetails = {
+  firstName: "Bonnie",
+  emailAddress: "bonny@cmil.mil",
+  rank: "CIV",
+  gender: "FEMALE",
+  country: "Albania",
+  endOfTourDate: ""
+}
+// need uniqueName due to saving previous ones as normal user
+export function createOnboardingNewPerson(personDetails, uniqueName) {
```

Comment:
Instead of doing this, I would have actually added the onboarding user with profile to the test data, and added it like this to the page and than use it in the tests. I think it simplifies the tests and avoids possible problems coming just from the creation workflow.

```
  static DEFAULT_CREDENTIALS = {
    user: "erin",
    superUser: "rebecca",
    adminUser: "arthur",
    onboardUser: "bonny",
userWithoutPosition: "noPosUser"
  }
```
But the code you wrote for the creation can be added as a test to the onBoardNewUser.

Reply:
I have tried that. The collisions occured, if you register an onboard user in one test, you bypass the onboard in other test

Reply:
Oh, you mean like, creating a user in the `insertBaseData.sql` file, which already bypasses the onboarding so we can test the home page directly. Yes, that would be a lot better, but don't know how to do that

Reply:
Yes.
I think it should work if you add to the insertBaseData.sql the following:
``` INSERT INTO people (uuid, name, status, role, emailAddress, phoneNumber, rank, biography, domainUsername, country, gender, endOfTourDate, createdAt, updatedAt)
  VALUES (lower(newid()), 'JACKSON, Jack', 0, 0, 'hunter+jack@dds.mil', '123-456-78960', 'OF-9', 'Jack is an advisor in EF 2.1', 'jack', 'Germany', 'MALE', DATEADD(year, 1, CURRENT_TIMESTAMP), CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);```
but then with other values. 
Add it to the advisors section of the file, if I remember it correctly new users are being made advisors by default?

If you don't also add a position for it, we have a user without a position ... at least I would expect it to.

Reply:
Yes, it worked.